packages/roosterjs-editor-plugins/lib/plugins/ImageSelection/ImageSelection.ts

```diff
@@ -1,18 +1,25 @@
 import { safeInstanceOf } from 'roosterjs-editor-dom';
```

Comment:
Should we add this plugin to the EditorOptions Pane plugin list?

Reply:
I'll add it